spec/integ/provisioning.spec.js

```diff
@@ -96,7 +96,7 @@ describe("Provisioning API", function() {
                     if (content.status === "failure") {
                         env.isFailed.resolve();
                     }
-                    else if (content.status == "success") {
+                    else if (content.status === "success") {
```

Comment:
in the future these should definitely be in a separate PR. These tests tend to be flaky and if they regress, it'd be hard to track down which PR did it.

Reply:
The ESLint `eqeqeq` rule became an error in a separate PR, which got merged after this one got opened.
https://github.com/matrix-org/matrix-appservice-irc/pull/1078/files#diff-1dc6ee56b778cd91e0327b52aaeaa1b9R87